test/acceptance/features/bindAppToService.feature

```diff
@@ -586,3 +585,86 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-backend-a" should be changed to "False"
         And Secret "binding-request-backend-a" contains "BACKEND_READY" key with value "true"
         And Secret "binding-request-backend-a" contains "BACKEND_HOST" key with value "example.common"
+
+        @negative1
+        Scenario: Service Binding without spec is not allowed in the cluster
+        When Invalid Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-without-spec
+            """
+        Then Error message "spec: Required value" is thrown
+        And Service Binding "binding-request-without-spec" is not persistent in the cluster
+
+        @negative1
+        Scenario: Service Binding with empty spec is not allowed in the cluster
+        When Invalid Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-empty-spec
+            spec:
+            """
+        Then Error message "invalid: spec: Invalid value: \"null\"" is thrown
+        And Service Binding "binding-request-empty-spec" is not persistent in the cluster
+
+        @negative1
+        Scenario: Service Binding with empty services is not allowed in the cluster
+        When Invalid Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-empty-services
+            spec:
+                services:
+            """
+        Then Error message "invalid: spec.services: Invalid value: \"null\"" is thrown
+        And Service Binding "binding-request-empty-services" is not persistent in the cluster
+
+        @negative1
+        Scenario: Service Binding without gvk of services is not allowed in the cluster
+        When Invalid Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-without-gvk
+            spec:
+                services:
+                -   name: backend-demo
+            """
+        Then Error message "spec.services.group: Required value" is thrown
+        And Error message "spec.services.kind: Required value" is thrown
+        And Error message "spec.services.version: Required value" is thrown
+        And Service Binding "binding-request-without-gvk" is not persistent in the cluster
+
+        @negative1
+        Scenario: Configuring exsting service binding with invalid spec configuration is not allowed
```

Comment:
better reads as "Removing services field from existing serivce binding is not allowed"

Reply:
Done.